vm/src/builtins/pystr.rs

```diff
@@ -3,6 +3,8 @@ use std::ops::Range;
 use std::string::ToString;
 use std::{char, ffi, fmt};
 
+use crossbeam_utils::atomic::AtomicCell;
+
```

Comment:
```suggestion
```

Reply:
I removed that useless blank line! Thanks for reviewing :)